templates/channels/channel_claim_line.haml

```diff
@@ -0,0 +1,59 @@
+-extends "smartmin/form.html"
+-load i18n compress temba
+
+-block title
+  .medium-help.icon-line.float-left
+
+  %h2.font_normalize.header-margin
+    -trans "Connect LINE Bot"
+
+-block pre-form
+  .info
+    %p
+      -blocktrans with name=brand.name
+        You can connect a <a href="https://developers.line.me/bot-api/overview" target="_blank">LINE Bot</a> to your {{ name }} account
+        to automate sending and receiving LINE messages.
+    %p
+      -blocktrans
+        You will need to <a href="https://developers.line.me/type-of-accounts/bot-api-trial" target="_blank">create a new LINE bot</a> and
+        get its Authentication Credentials. To do so:
+
+      %ol.line-steps
+        %li
+          -blocktrans
+            Access the page <a href="https://developers.line.me/channels/" target="_blank">LINE Channels</a> e add a new Channel.</li>
+
+        %li
+          -blocktrans
+            After add, select your Channel, go to basic information and click EDIT.
+
+        %li
+          -blocktrans
+            Enter the callback URL in the Callback URL section (Generated by RapidPRO after this step).
+
```

Comment:
Language is a bit goofy through this whole section. Also make sure to reference {{brand}} instead of RapidPro.